### PR TITLE
fix(chart): support prerelease with kubeVersion for AWS EKS clusters

### PR DESCRIPTION
**Pull request by jgrasl**
AWS EKS clusters has a -eks in their version numbers for example v1.28.11-eks-db838b0 which helm interprets as a prerelease version becasue of the dash after the semver version. It explained in detail here: [https://github.com/helm/helm/issues/10375](url)
Adding -0 to the kubeVersion 1.21.0 makes it possible to deploy the helm chart on EKS clusters.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,13 @@
 #### Fixed
 #### Removed
 
+### [Unreleased](https://github.com/redpanda-data/helm-charts/releases/tag/redpanda-FILLMEIN) - YYYY-MM-DD
```
Comment (RafalKorepta) [inline CHANGELOG.md:11]:
```suggestion
### [0.7.28](https://github.com/redpanda-data/helm-charts/releases/tag/console-0.7.28) - 2024-08-08
```

Comment (RafalKorepta) [reply CHANGELOG.md:11]: P.S. Look at the Github you probably need to rebase `CHANGELOG.md`

**Review by RafalKorepta — APPROVED** [review]

**Comment by jgrasl** [comment]
> @jgrasl Could you run `nix develop -c go test ./charts/console -short -update` to update our golden test files?

Done. Updated the testdata

**Comment by RafalKorepta** [comment]
I'm sorry. There is one additional update
```
nix develop -c task ci:lint
```

**Comment by jgrasl** [comment]
> I'm sorry. There is one additional update
> 
> ```
> nix develop -c task ci:lint
> ```
> 
> As per error in the CI https://github.com/redpanda-data/helm-charts/actions/runs/10305205098/job/28526760104?pr=1480#step:5:293
> 
> ```
> diff --git a/charts/console/README.md b/charts/console/README.md
> index 87b89a8..b39c6dd 100644
> --- a/charts/console/README.md
> +++ b/charts/console/README.md
> @@ -3,7 +3,7 @@
>  description: Find the default values and descriptions of settings in the Redpanda Console Helm chart.
>  ---
>  
> -Version: 0.7.27 Type: application AppVersion: v2.7.0
> +Version: 0.7.28 Type: application AppVersion: v2.7.0
>  
>  This page describes the official Redpanda Console Helm Chart. In particular, this page describes the contents of the chart’s [`values.yaml` file](https://github.com/redpanda-data/helm-charts/blob/main/charts/console/values.yaml).
>  Each of the settings is listed and described on this page, along with any default values.
> @@ -21,7 +21,7 @@ Autogenerated from chart metadata using [helm-docs v1.13.1](https://github.com/n
>  
>  ## Requirements
>  
> -Kubernetes: `>= 1.21`
> +Kubernetes: `>= 1.21.0-0`
>  
>  ## Settings
> ```

I ran it and updated the README.md

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,13 @@
 #### Fixed
 #### Removed
 
+### [0.7.28](https://github.com/redpanda-data/helm-charts/releases/tag/console-0.7.28) - 2024-08-08
+#### Added
+#### Changed
+#### Fixed
+* Fixed kubeVersion to be able to deploy on AWS EKS clusters.
+#### Removed
+
```
Comment (RafalKorepta) [inline CHANGELOG.md:17]: Could this section be moved to 
https://github.com/redpanda-data/helm-charts/blob/b35531cf8de4b173024e12b697ba8cce678a7c27/CHANGELOG.md?plain=1#L130
?

I didn't realised this section is for Redpanda chart not console.

Comment (jgrasl) [reply CHANGELOG.md]: For sure, sorry didn't see there is different sections in the changelog. Moved it to the console section.

**Pull request merged**